src/components/home/elements/NavBar.vue

```diff
@@ -166,4 +263,24 @@ export default {
 .nav-item {
     background-color: rgb(240, 240, 240);
 }
+.block {
+    position: relative;
+    display: inline-block;
+    margin-top:15px;
+    margin-right: 10px;
+    height:19px;
+    color: rgba(0, 0, 0, 0.5);
+}
+.block .time-height{
+    height:19px;
+    top: 15px;
+    right: 0;
+    color: rgba(0, 0, 0, 0.5);
+    position: absolute;
```

Comment:
I think remove this `position: absolute;` can make UI more nice look. All info written in one line.